_posts/zh/newsletters/2019-06-05-newsletter.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Bitcoin Optech Newsletter #49'
+permalink: /zh/newsletters/2019/06/05/
+name: 2019-06-05-newsletter-zh
+slug: 2019-06-05-newsletter-zh
+type: newsletter
+layout: newsletter
+lang: zh
+---
+本周的 Newsletter 描述了提议的 Erlay 协议，该协议可以显著减少节点之间中继未确认交易的开销，总结了 Bitrefill CEO Sergej Kotliar 关于闪电网络的执行简报，并列出了一些最近对上周描述的 COSHV 提案的更改内容。还包括我们关于 Bech32 发送支持和流行比特币基础设施项目中值得注意的代码更改的常规部分。
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+{% include specials/2019-exec-briefing/zh/lightning.md %}
+
+## 行动项
+
+*本周无。*
+
+## 新闻
+
+- **<!--erlay-proposed-->****Erlay 提议**：Gleb Naumenko、Pieter Wuille、Gregory Maxwell、Sasha Fedorova 和 Ivan Beschastnikh 的一篇新论文描述了一种替代交易中继协议，[Erlay][]。当前，当一个节点了解到一个新的交易并且该交易通过了其中继策略时，它会将该交易的 txid 通知其所有的节点（除了之前已经通知过的节点）。这是一种非常简单有效的策略，但效率不高——节点接收到的大多数通知是它几分钟前从其他节点已经得知的交易通知。根据论文，这种冗余浪费了大约 44% 的所有节点带宽。
```

Comment:
Your items are using a different format that other newsletters which could be causing the issue.

```suggestion
- **<!--erlay-proposed-->****Erlay 提议: **Gleb Naumenko、Pieter Wuille、Gregory Maxwell、Sasha Fedorova 和 Ivan Beschastnikh 的一篇新论文描述了一种替代交易中继协议，[Erlay][]。当前，当一个节点了解到一个新的交易并且该交易通过了其中继策略时，它会将该交易的 txid 通知其所有的节点（除了之前已经通知过的节点）。这是一种非常简单有效的策略，但效率不高——节点接收到的大多数通知是它几分钟前从其他节点已经得知的交易通知。根据论文，这种冗余浪费了大约 44% 的所有节点带宽。
```